frontend/src/types/record.ts

```diff
@@ -0,0 +1,30 @@
+export type AlgoliaRecord = {
+  objectID: string;
+
+  url: string;
+  origin: string;
+  title: string;
```

Comment:
```suggestion
  title?: string;
```

Reply:
title can not be empty (even if it was set like that in the original types) the code will always output at least `""` (empty string)

Reply:
Which "code"?

Reply:
cleanWhitespaces()